app/components/general/components/bc-table/macro.njk

```diff
@@ -7,10 +7,19 @@
 {% macro bcTable(params) %}
   <div data-test-id="table">
     {% if params.columnInfo %}
-      <div data-test-id="table-headings" class="nhsuk-u-font-size-19 nhsuk-grid-row nhsuk-u-margin-1 nhsuk-u-padding-bottom-3 bc-t-border-bottom--grey-3">
+      <div data-test-id="table-headings" class="nhsuk-u-font-size-19 nhsuk-grid-row nhsuk-u-margin-1 bc-t-border-bottom--grey-3">
```

Comment:
@alibombaye pls can you check in admin that removing this padding bottom doesn't affect the tables in there

Reply:
ok I'll npm link it and see what happens

Reply:
it looks fine as is